CHANGELOG.md

```diff
@@ -23,6 +24,7 @@
 - Move SentryExperimentalOptions to a property defined in Swift (#5329)
 - Add redaction in session replay for `SFSafariView` used by `SFSafariViewController` and `ASWebAuthenticationSession` (#5408)
 - Convert SentryNSURLRequest to Swift (#5457)
+- [Structured Logging] Rename SentryLog to SentrySDKLog ([#5473](https://github.com/getsentry/sentry-cocoa/pull/5473))
```

Comment:
`m`: I think this is only an internal refactoring. I don't think this requires a changelog entry.